pkg/orchestrator/orchestrator.go

```diff
@@ -134,6 +134,10 @@ func container(cr *apiv2.PerconaServerForMySQL) corev1.Container {
 				Name:  "MYSQL_SERVICE",
 				Value: mysql.ServiceName(cr),
 			},
+			{
+				Name:  "RAFT_ENABLED",
+				Value: "false",
+			},
```

Comment:
I was following the orchestrator logs and there were too much noise because of raft errors. If it's OK I want to keep it here and have a separate PR for proper Orchestrator raft setup

Reply:
please create the ticket

Reply:
https://jira.percona.com/browse/K8SPS-32